platform/src/main/java/org/hillview/sketches/FullCorrelationSketch.java

```diff
@@ -0,0 +1,81 @@
+package org.hillview.sketches;
+
+import org.hillview.dataset.api.ISketch;
+import org.hillview.table.api.*;
+import org.hillview.utils.Converters;
+
+import javax.annotation.Nullable;
+import java.security.InvalidParameterException;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class is very similar to the SampleCorrelationSketch, apart from two important differences. First, it uses
```

Comment:
This comment should be kept, but please add a comment that describes what this is really doing.
The user should not need to browse the other class to understand this one. Maybe you can just steal some of the comments from the other class.